controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/configserver/Node.java

```diff
@@ -495,12 +503,17 @@ public Builder openStackId(String openStackId) {
             return this;
         }
 
+        public Builder switchHostname(Optional<String> switchHostname) {
```

Comment:
I don't think builders should take Optionals, instead the caller should avoid calling if the field is not present.